src/frontend/src/service/api-service.test.ts

```diff
@@ -52,8 +52,11 @@ describe('API SERVICE TEST', () => {
     };
     apiService(mockStore.dispatch, request).then((response: any) => {
       expect(response).toEqual({
-        data: fakeRecord,
-        errors: []
+        data: {
+          data: {
+            record: fakeRecord
```

Comment:
The backend is being wrapped in two data objects?

Reply:
Once explicitly by us at https://github.com/codeforpdx/recordexpungPDX/blob/master/src/backend/expungeservice/endpoints/search.py#L50 and then once by I believe Flask.

Reply:
Understood.